js/clinicalTimeline.js

```diff
@@ -957,7 +776,7 @@ var clinicalTimeline = (function(){
    * Return zoomFactor by specifying what kind of zoomLevel on the x axis (e.g.
    * years, days) is desired
    */
-  function getZoomFactor(zoomLevel, minDays, maxDays, width) {
+  timeline.getZoomFactor = function(zoomLevel, minDays, maxDays, width) {
```

Comment:
Since we now also have a `setZoomFactor` it's a little weird to have `getZoomFactor` not just return a variable on `timeline`. You might want to change this to `computeZoomFactor` and let `getZoomFactor` just return the variable on timeline.